sympy/utilities/lambdify.py

```diff
@@ -17,13 +17,24 @@
 
 __doctest_requires__ = {('lambdify',): ['numpy', 'tensorflow']}
 
+# These are the namespaces the lambda functions will use.
+MATH = {}
+MPMATH = {}
+NUMPY = {}
+SCIPY = {}
+TENSORFLOW = {}
+TORCH = {}
+SYMPY = {}
+NUMEXPR = {}
```

Comment:
These lines are overridden by the lines below https://github.com/sympy/sympy/pull/15484/files#diff-16366e978f6ce314c0fa904081d58c24R45-R51